src/components/Shared/Header/ListHeader.test.tsx

```diff
@@ -226,6 +225,33 @@ describe('ListHeader', () => {
 
       expect(getByPlaceholderText('Search Tasks')).toBeVisible();
     });
+    it('Should update count upon deletion', async () => {
+      const { queryByText, getByText } = render(
+        <Components
+          selectedIds={['a', 'b', 'c']}
+          page={PageEnum.Task}
+          totalItems={50}
+          showShowingCount={true}
+        />,
+      );
+      expect(queryByText('Showing 50')).toBeInTheDocument();
```

Comment:
If we're expecting the element to exist, use `getBy...` instead of `queryBy...`. `queryBy` is only for when elements *don't* exist.